contract/system/vote.go

```diff
@@ -75,38 +75,62 @@ func newVprCmd(ctx *SystemContext, vr *VoteResult) *vprCmd {
 		}
 	} else {
 		cmd.add = func(v *types.Vote) error {
-			return cmd.addVote(v)
+			cmd.addVprWithHandleException(v) // calculate voting power rank with handle exception
```

Comment:
"with handle exception" is not proper english language

it can be "with exception handling" or "and handle exceptions"

Reply:
the function name can be renamed to just `cmd.addVpr()`

Reply:
ok. i`ll rename it